polymetis/polymetis/tests/python/polysim/test_env.py

```diff
@@ -55,6 +56,46 @@
     },
 )
 
+franka_panda_abs = OmegaConf.create(
+    {
+        "robot_description_path": "/root/project/polymetis/polymetis/data/franka_panda/panda_arm.urdf",
```

Comment:
Yeah... let's not hardcode the directory structure of a random circleci image into our unit tests. This is pretty much guaranteed to fail locally.